docs/adoc/howtos/Chart-HowTo.adoc

```diff
@@ -0,0 +1,181 @@
+
+==== Minimal Code for a New Chart
+
+The chart is created using an `AbstractChartField`  and placed inside a `GroupBox`.
+
+[source,java]
+----
+public class ChartField extends AbstractChartField<Chart> {
+  public class Chart extends AbstractChart {
+  }
+}
+----
+
+==== Add data to the chart
+
+The ice cream shop has sold the following amount of scoops:
+
+.Table Scoops per month and flavor
+|===
+| |Jan. |Feb. |Mar. |Apr. |May |Jun. |Jul. |Aug. |Sept. |Oct. |Nov. |Dec.
+
+|Vanilla
+|0
+|0
+|0
+|94
+|162
+|465
+|759
+|537
+|312
+|106
+|0
+|0
+
+|Chocolate
+|0
+|0
+|0
+|81
+|132
+|243
+|498
+|615
+|445
+|217
+|0
+|0
+
+|Strawberry
+|0
+|0
+|0
+|59
+|182
+|391
+|415
+|261
+|75
+|31
+|0
+|0
+|===
+
+We create a data object and pass it to the chart.
+
+[source,java]
+----
+ChartData data = new ChartData();
+
+List<IChartAxisBean> axis = new ArrayList<>();
+Stream.of("Jan.", "Feb.", "Mar.", "Apr.", "May", "Jun.", "Jul.", "Aug.", "Sept.", "Oct.", "Nov.", "Dec.")
+    .forEach(label -> axis.add(new ChartAxisBean(label, label)));
+
+data.getAxes().add(axis);
+
+MonupleChartValueGroupBean vanilla = new MonupleChartValueGroupBean();
+vanilla.setGroupName("Vanilla");
+IntStream.of(0, 0, 0, 94, 162, 465, 759, 537, 312, 106, 0, 0)
+    .forEach(value -> vanilla.getValues().add(new BigDecimal(value)));
+data.getChartValueGroups().add(vanilla);
+
+MonupleChartValueGroupBean chocolate = new MonupleChartValueGroupBean();
+chocolate.setGroupName("Chocolate");
+IntStream.of(0, 0, 0, 81, 132, 243, 498, 615, 445, 217, 0, 0)
+    .forEach(value -> chocolate.getValues().add(new BigDecimal(value)));
+data.getChartValueGroups().add(chocolate);
+
+MonupleChartValueGroupBean strawberry = new MonupleChartValueGroupBean();
+strawberry.setGroupName("Strawberry");
+IntStream.of(0, 0, 0, 59, 182, 391, 415, 261, 75, 31, 0, 0)
+    .forEach(value -> strawberry.getValues().add(new BigDecimal(value)));
+data.getChartValueGroups().add(strawberry);
+
+chart.setData(data);
+----
+
+The chart will now look like this:
+
+image::{imgsdir}/pieChartDefault.png[]
+
+It looks like this, because the default type is `pie`, the default value of `maxSegments` is `5` and the first three segments in each dataset are 0.
+
+==== Chart configuration
+
+Let's change it to a bar chart and use another color scheme:
+
+[source,java]
+----
+IChartConfig config = BEANS.get(IChartConfig.class)
+    .withType(IChartType.BAR)
+    .withColorScheme(ColorScheme.RAINBOW);
+
+chart.setConfig(config);
+----
+
+image::{imgsdir}/barChartRainbow.png[]
+
+Now we add labels to the scales and set some custom colors.
+
+[source,java]
+----
+vanilla.setColorHexValue("#fdf2d1");
+chocolate.setColorHexValue("#94654c");
+strawberry.setColorHexValue("#f89fa1");
+
+config.withAutoColor(false)
+    .withXAxisLabelDisplay(true)
+    .withXAxisLabel("Month")
+    .withYAxisLabelDisplay(true)
+    .withYAxisLabel("Scoops");
+
+chart.setData(data);
+chart.setConfig(config);
+----
+
```

Comment:
Could you mention that it is also possible to style the Charts using CSS even though they are rendered on a canvas and give an example how to do that?